connector/dialect.tdd

```diff
@@ -286,6 +286,70 @@
       <argument type='real' />
       <argument type='int' />
     </function>
+    <function group='numeric' name='LN' return-type='real'>
+      <formula>LOG(%1, 2.718281828459046)</formula>
```

Comment:
Similar to `PI`, is this constant "good enough" to give correct results in all cases?

Reply:
That's a good question, I'll take a look at what the TDVT expects in the test and see if it matches what is returned from mongodb.

Reply:
I compared the results from mongodb with `$log : [ %1, 2.718281828459046 ]` to the TDVT results and the results from mongodb have more precision whereas the TDVT results seem rounded such as:
LN(3)
mongodb: 1.0986122886681096    TDVT: 1.0986123
LN(2)
mongodb: 0.6931471805599452    TDVT: 0.69314718
LN(15.7)
mongodb: 2.753660712354262      TDVT: 2.7536607

It seems then that the value for `e` produces an accurate result for TDVT.   As for the precision differences, we may have to get the test to fully run to see whether the test framework is rounding the value when doing the comparison checking.

Reply:
Ok that sounds good. Looks like the constant we're using is pretty decent, so hopefully we will be able to just stick with it 🤞